README.md

```diff
@@ -42,6 +42,15 @@ options. When used with Brim, you should typically use the `brimcap` binary
 found in Brim's `zdeps` directory (as described in the article), since this
 version should be API-compatible with that version of Brim and its Zed backend.
 
+## Brimcap Queries
+
+Included in this repo is a `queries.json` file with some helpful queries for getting
```

Comment:
```suggestion
Included in this repo is a [`queries.json`](./queries.json?raw=1) file with some helpful queries for getting
```

I figure hyperlinking to the raw version of the queries file might help a little bit.

I've always had mixed feelings about the move of the prior query library to the Brimcap repo, though given the direction of the app I understand why it's justified and I'm prepared to go with it. I see a little devil in the details though. While _we_ know the direction the app is going, the fact remains that the vast majority of our existing users are still ones that came for the security use cases and they may not be keeping up with our strategic vision, so yanking back something they may have come to depend on might be somewhat jarring. Also, since many of the users were effectively app-only, many of them may not be super GitHub savvy (picture a non-developer Windows user, for instance). So while a README that speaks of using a file from a repo is totally intuitive to folks like us, the same may not be true for all of our users. Finally, I know how trying to right-click download files from the GitHub web interface can be frustrating for some since getting to the "raw" version is not always obvious to new users, so I'm hoping putting a hyperlink directly to the raw version might help them along.

If I had my wish, what I'd love would be if the app's query library had an option to download from a URL in addition to its current drag-&-drop. That way we could Release Note something with Zui v1.0 that shows them the raw Brimcap file URL they'd cut & paste into the app and they're done. GIven our limited Dev resources I know this may be too fancy to justify now. However I know that we've discussed at times wanting to make it easy for users to share this kind of stuff via Gists and the like (#1421), make it easy for users to know when a query lib they previously imported has evolved and make it easy for them to re-sync it (#1422), etc. So I can't help but wish we could the base level network-based download in sooner to make this transition as easy as possible.

Ok, enough with the soapbox. I still approve the current approach as reasonable progress. 👍